File: beginner/pagina01/1009/1009.ts
const fs = require('fs')
const input = fs.readFileSync('/dev/stdin', 'utf-8')
const lines = input.split('\n')

lines.shift() //nnumero do empregado não é relevante

let salary = parseFloat(lines.shift())
let sales = parseFloat(lines.shift())

let Bonus = ((sales * 15) / 100)
let salaryWithBonus = salary + Bonus

console.log(`TOTAL = R$ ${salaryWithBonus.toFixed(2)}`)

File: beginner/pagina01/1009/1009.cs
using System;
using System.Globalization;

public class URI
{
    public static void Main(string[] args)
    {
        string name = Console.ReadLine();

        double salary = double.Parse(Console.ReadLine());
        double sales = double.Parse(Console.ReadLine());

        double bonus = (sales * 15)/ 100;

        salary += bonus;

        Console.WriteLine("TOTAL = R$ {0}", salary.ToString("F2", CultureInfo.InvariantCulture));
    }
}